lib/SILOptimizer/Transforms/ArrayElementValuePropagation.cpp

```diff
@@ -249,37 +311,73 @@ bool ArrayAllocation::collectForwardableValues() {
 namespace {
 
 class ArrayElementPropagation : public SILFunctionTransform {
+
 public:
   ArrayElementPropagation() {}
 
   StringRef getName() override {
     return "Array Element Propagation";
   }
 
+  void replaceAppendCalls(SmallVector<ValueReplacementsPair, 4> Repls) {
+    auto &Fn = *getFunction();
+    auto &M = Fn.getModule();
+    auto &Ctx = M.getASTContext();
+
+    if (Repls.empty())
+      return;
+
+    llvm::dbgs() << "Array append contentsOf calls replaced in "
```

Comment:
Put this debug output into `DEBUG`, i.e. `DEBUG(your_code);`, otherwise it will be printed unconditionally on every compilation.